### PR TITLE
Fix js docs in `ads` and `3p` folder

### PR DESCRIPTION
**Pull request by prateekbh**
- Fixes jsdocs

### CODE REVIEW

**Comment by prateekbh** [comment]
@rsimha This is the last PR for fixing jsDocs. After this PR will a revse patch of https://github.com/ampproject/amphtml/pull/16339/ will suffice or will it require something else than that to just enable the rules back?

**Comment by rsimha** [comment]
@prateekbh To answer your question, after this PR is merged, all we need to do is change the rules below from warning mode (`1`) to error mode (`2`).

In fact, you could probably do so as part of this PR as a way to make sure you fixed all instances.

https://github.com/ampproject/amphtml/blob/34b4e16fd2cdc7b9776e33e6d0f87738a644827d/.eslintrc#L92-L98

https://github.com/ampproject/amphtml/blob/34b4e16fd2cdc7b9776e33e6d0f87738a644827d/.eslintrc#L147-L155

**Comment by prateekbh** [comment]
@rsimha requesting for a second look for the changes made to `.eslintrc`

File: .eslintrc [hunk 1]
```diff
@@ -170,9 +170,16 @@
   },
   "overrides": [
     {
-      "files": ["test/**/*.js", "extensions/**/test/**/*.js"],
+      "files": ["test/**/*.js", "extensions/**/test/**/*.js", "ads/**/test/**/*.js", "testing/**/*.js"],
```
Comment (rsimha) [inline .eslintrc]: Cool!

File: .eslintrc [hunk 2]
```diff
@@ -89,13 +89,13 @@
...
-    "jsdoc/require-param-name": 1,
-    "jsdoc/require-param-type": 1,
-    "jsdoc/require-returns-type": 1,
+    "jsdoc/check-param-names": 2,
+    "jsdoc/check-tag-names": 2,
+    "jsdoc/check-types": 2,
+    "jsdoc/require-param": 2,
+    "jsdoc/require-param-name": 2,
+    "jsdoc/require-param-type": 2,
+    "jsdoc/require-returns-type": 2,
```
Comment (rsimha) [inline .eslintrc]: Does `gulp lint` now work across the entire codebase with these new settings? Kudos to you for making it happen!

Comment (prateekbh) [reply .eslintrc]: So after the change there were error from testing folders, which I have ignored in the settings.
If that answers your question.

File: .eslintrc (hunk 1 above)
Comment (rsimha) [reply .eslintrc]: I wonder if this line can be simplified to:
```
     "files": ["**/test/**/*.js", "testing/**/*.js"],
```

**Pull request merged**